crates/codegen/parser/generator/src/code_generator.rs

```diff
@@ -30,6 +30,8 @@ pub struct CodeGenerator {
     rule_kinds: BTreeSet<&'static str>,
     token_kinds: BTreeSet<&'static str>,
     production_kinds: BTreeSet<&'static str>,
+    // TODO: Make it lexical context-dependent
+    trivia_kinds: BTreeSet<&'static str>,
```

Comment:
Should I leave this for later when we separate trivia for contexts or should I do it here as part of this work?

Reply:
I think we should leave this, because one trivia kind will do for Solidity.

Reply:
Done in 458f8afb35c52bbd1ef6eab2f45e2fdf80bfed88